include/zephyr/rtio/rtio.h

```diff
@@ -1175,6 +1175,32 @@ static inline void rtio_cqe_release(struct rtio *r, struct rtio_cqe *cqe)
 	rtio_cqe_pool_free(r->cqe_pool, cqe);
 }
 
+/**
+ * @brief Flush completion queue
+ *
+ * @param r RTIO context
+ * @return The operation completion result
+ * @return 0 if the queued operations completed with no error
+ * @return <0 on error
```

Comment:
```suggestion
 * @retval 0 if the queued operations completed with no error
 * @retval <0 on error
```

Reply:
(and yes, Doxygen guidelines are coming, since I know there's a lot to remember :| https://github.com/zephyrproject-rtos/zephyr/pull/93539 -- Thanks!)

Reply:
OK, thanks. I copied the way I found in rtio_cqe_get_mempool_buffer().
But the way suggested by you makes much more sense to me.